portfolio/portfolio_test.go

```diff
@@ -501,147 +275,129 @@ func TestGetPortfolioSummary(t *testing.T) {
 		return Coin{}
 	}
 
-	if !getTotalsVal(currency.LTC).Coin.Equal(currency.LTC) {
-		t.Error("mismatched currency")
-	}
-
-	if getTotalsVal(currency.ETH).Coin.Equal(currency.LTC) {
-		t.Error("mismatched currency")
-	}
-
-	if getTotalsVal(currency.LTC).Balance != 23 {
-		t.Error("incorrect balance")
-	}
-
-	if getTotalsVal(currency.BTC).Balance != 200 {
-		t.Error("incorrect balance")
-	}
+	assert.Equal(t, currency.LTC, getTotalsVal(currency.LTC).Coin, "Coin should be LTC")
+	assert.Equal(t, 23.0, getTotalsVal(currency.LTC).Balance, "LTC balance should be correct")
+	assert.Equal(t, 200.0, getTotalsVal(currency.BTC).Balance, "BTC balance should be correct")
+	assert.Equal(t, 69.0+420.0+42, getTotalsVal(currency.ETH).Balance, "ETH balance should be correct")
 }
 
-func TestGetPortfolioGroupedCoin(t *testing.T) {
+func TestGetPortfolioAddressesGroupedByCoin(t *testing.T) {
```

Comment:
🐒  Adding this as a in-code placeholder for the review summary comment that I think we should replace `Coin` with `Currency` throughout
```suggestion
func TestGetPortfolioAddressesGroupedByCurrency(t *testing.T) {
```